packages/react/navigation-menu/src/NavigationMenu.tsx

```diff
@@ -915,17 +914,24 @@ const NavigationMenuViewportImpl = React.forwardRef<
     CONTENT_NAME,
     props.__scopeNavigationMenu
   );
-  const [activeContent, setActiveContent] = React.useState<NavigationMenuContentElement | null>(
-    null
-  );
-  const contentSize = useSize(activeContent);
-  const viewportWidth = contentSize ? contentSize?.width + 'px' : undefined;
-  const viewportHeight = contentSize ? contentSize?.height + 'px' : undefined;
+  const [size, setSize] = React.useState<{ width: number; height: number } | null>(null);
+  const [contentNode, setContentNode] = React.useState<NavigationMenuContentElement | null>(null);
```

Comment:
Renamed to avoid awkward wrapping, if we prefer the originals then that's fine by me, I just found it irritating.

Reply:
To be honest in most component I think we call these just `[content, setContent]`.